redaxo/src/core/assets/standard.js

```diff
@@ -785,3 +785,8 @@ function rex_searchfield_init(selector_id) {
             .trigger('propertychange').focus();
     });
 }
+
+// tooltip toggle on title
+$(document).on('rex:ready', function (event, container) {
+     $('[data-toggle="tooltip"]').tooltip();
```

Comment:
Innerhalb von `rex:ready` sollte man immer mit dem `container` arbeiten.
Ansonsten würden Tooltips außerhalb des pjax-containers bei jeder pjax-Aktion erneut initialisiert werden.

```suggestion
     container.find('[data-toggle="tooltip"]').tooltip();
```

Reply:
ah ok ... gut zu wissen